andela/ah-leagueOfLegends-frontend
# #159965439 Login

#### What does this PR do?
Allows users to login into the application

#### Description of task to be completed
Present users with a login form where they input their email and password and then redirect them to the home page

#### How should this be tested manually?
* Clone the repo and checkout ft-login
* `npm install` to install all dependancies
* start the server `npm start`
* In the browser, click the signin button in the navbar
* A signin modal should be shown

#### What are the relevant PT stories?
[#159965439](https://www.pivotaltracker.com/story/show/159965439)

#### Screenshots
<img width="1280" alt="screen shot 2018-10-13 at 16 40 41" src="https://user-images.githubusercontent.com/7246507/46906100-c95f0080-cf06-11e8-8c85-b05affec417d.png">

<img width="1280" alt="screen shot 2018-10-13 at 16 40 52" src="https://user-images.githubusercontent.com/7246507/46906103-d7148600-cf06-11e8-8299-eb3e07c7e98d.png">


<img width="1280" alt="screen shot 2018-10-13 at 16 39 37" src="https://user-images.githubusercontent.com/7246507/46906087-a896ab00-cf06-11e8-83d5-c9f1df100518.png">
